dts/bindings/led/ti,lp5569.yaml

```diff
@@ -5,6 +5,11 @@ compatible: "ti,lp5569"
 include: i2c-device.yaml
 
 properties:
+  enable-gpios:
```

Comment:
I wonder if it would not be better to add a fixed regulator in the devicetree configuration of your board... If not, this addition also needs to go in the release notes.

Reply:
Since this pin is a dual-purpose enable- and PWM gpio on the chip, that also has some specific timings tied to it, I think it makes sense to describe the pin in the lp5569 nodes.

But functionally, there'd be no issue in tying the enabled part to a regulator with the specific timing described.

Reply:
You are right.